cpp/merge_print_and_online.cc

```diff
@@ -278,7 +278,7 @@ const std::set<MARC::Tag> LINK_TAGS{ "800", "810", "830", "773", "775", "776" };
 // Make inferior works point to the new merged superior parent found in "ppn_to_canonical_ppn_map".  Links referencing a key in
 // "ppn_to_canonical_ppn_map" will be patched with the corresponding value.
 // only 1 uplink of the same tag type will be kept.  Also some cross links will be patched.
-unsigned PatchUpAndCrossLinks(MARC::Record * const record, const std::unordered_map<std::string, std::string> &ppn_to_canonical_ppn_map) {
+unsigned PatchUpUplinksAndCrossLinks(MARC::Record * const record, const std::unordered_map<std::string, std::string> &ppn_to_canonical_ppn_map) {
```

Comment:
UpUp? Intended?

Reply:
Absolutely!  Read it as "patch up uplinks and cross links"!